phabricatoremails/cli.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import pathlib
+
+import sentry_sdk
+from phabricatoremails.migrate import migrate
+from phabricatoremails.settings import Settings
+from phabricatoremails.init import init
+from phabricatoremails.service import service
+from statsd import StatsClient
+
+
+def parse_command():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--settings-file",
+        default="settings.ini",
```

Comment:
I'm allowing the user to pass in a custom path for `settings.ini` based on [this conversation](https://github.com/mozilla-conduit/phabricator-emails/pull/1#discussion_r402718884) in which `glob` recommended against forcing the use of the working directory to locate files.

I'll create `settings.development.ini` and `settings.production.ini` files. I think that a config generator could be useful, but I'll defer that to the future (it's use case is mostly solved by developers copying `settings.development.ini` to `settings.ini`). Since we still will want a single point of documentation for this INI file, I'll still need to have a more generic version shown in the README.

Reply:
I think the context of the two conversations is a little different. I'm not suggesting the use of the current working directory for this -- I'm suggesting using a fixed defined path rather than users entering an arbitrary path or string. The default here is still using the current working directory by the way (i.e. `pathlib.Path("settings.ini")`).

You could also use `BASE_PATH` here, once it is defined.

Reply:
> You could also use BASE_PATH here, once it is defined.

Oh, I think I see the missing information here: `phabricator-emails` isn't packaged with a `settings.ini`. Rather, the configuration is defined by the environment in which it's run in. Therefore, the settings wouldn't necessary be stored in the same location as the source code.

> The default here is still using the current working directory by the way (i.e. pathlib.Path("settings.ini")).

Right, the issue from earlier is that we shouldn't **force** the use of the working directory to locate files. I've defaulted to using the working directory, but now it is also flexible enough to allow providing a custom path.

Reply:
This conversation is missing an update: we can't parameterize the `settings.ini` location on the CLI, because Alembic cannot be (nicely) configured to look in a custom location for DB information. So, we always look in the package directory for `settings.ini` now.